### ISSUE TITLE
Adding to the top level

### ISSUE THREAD

**Issue by rendix2** [stefangabos/Zebra_Mptt#8]
Hello,

if i am adding node to top level (without parent), calculation of left and right values is bad.  But not for first time. It hapends if you add root, some nodes under root and then you add another node without parent.  In this case the values are bad. I suggest add something like this after line 188

`if ($boundary === 0) {

INSERT INTO tree (lft, rgt, title) SELECT IFNULL(MAX(rgt), 0) + 1, IFNULL(MAX(rgt), 0) + 2, title FROM tree
}`

**Comment by stefangabos** [comment]
I am unable to reproduce that. Can you show me the PHP code from which the bug results?
I've tried something like
```php
$first = $mptt->add(0, 'First top level');
$mptt->add($first, 'Some entry');

$second = $mptt->add(0, 'Second top level');
$mptt->add($second, 'Some entry');

$third = $mptt->add(0, 'Third top level');
$mptt->add($third, 'Some entry');
```

**Comment by stefangabos** [comment]
i don't see anything wrong with the result...

**Comment by stefangabos** [comment]
i used 
```print_r('<pre>');
print_r($mptt->get_tree());
```
after your code

**Comment by rendix2** [comment]
yes, tree is printed well, but right values are bad. I think i had two nodes with same right values.

**Comment by stefangabos** [comment]
in your example everything looks fine and left and right values are ok. i can only fix it if i am able to reproduce it...

**Comment by stefangabos** [comment]
Another user reported something similar and it turns out that he was using a modified table structure where the custom columns were missing a `default` value. In MySQL previous to 5.7.1 apparently there was an issue where, when doing this, MySQL would [die with an error](https://stackoverflow.com/questions/15438840/mysql-error-1364-field-doesnt-have-a-default-values)  Additionally, my library was silently failing in these scenarios because of the way it used LOCK TABLEs.

I fixed things on my side but if you are using a custom table structure, make sure the new columns have default values.

Thanks!
